trace/kafka/async/option.go

```diff
@@ -0,0 +1,41 @@
+package kafka
```

Comment:
Why the need for option if you have a builder?

Reply:
I am afraid this is a remnant of the original PR. I will remove it.